### PR TITLE
DevTools 105 > Recorder > Breakpoints and step through

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/blog/new-in-devtools-105/#recorder

Also, moved a related section to features ref to streamline the tutorial.

Publish on Aug 30

### CODE REVIEW

File: site/en/docs/devtools/recorder/reference/index.md [hunk 1]
```diff
@@ -68,11 +68,11 @@ You can do the following with each export option:
 
 You can install a Chrome extension to export replay scripts in your favorite format. 
 
-For example, the [WebPageTest extension](https://chrome.google.com/webstore/detail/webpagetest-recorder-exte/eklpnjohdjknellndlnepihjnhpaimok) let’s you export user flows from the Recorder directly as [WebPageTest Custom scripts](https://docs.webpagetest.org/scripting/). See [Converting user flows to WebPageTest custom scripts](https://blog.webpagetest.org/posts/introducing-the-new-webpagetest-recorder-chrome-extension/) to learn more. 
+For example, the [WebPageTest extension](https://chrome.google.com/webstore/detail/webpagetest-recorder-exte/eklpnjohdjknellndlnepihjnhpaimok) let's you export user flows from the Recorder directly as [WebPageTest Custom scripts](https://docs.webpagetest.org/scripting/). See [Converting user flows to WebPageTest custom scripts](https://blog.webpagetest.org/posts/introducing-the-new-webpagetest-recorder-chrome-extension/) to learn more.
```
Comment (rachelandrew) [inline site/en/docs/devtools/recorder/reference/index.md:71]:
```suggestion
For example, the [WebPageTest extension](https://chrome.google.com/webstore/detail/webpagetest-recorder-exte/eklpnjohdjknellndlnepihjnhpaimok) lets you export user flows from the Recorder directly as [WebPageTest Custom scripts](https://docs.webpagetest.org/scripting/). See [Converting user flows to WebPageTest custom scripts](https://blog.webpagetest.org/posts/introducing-the-new-webpagetest-recorder-chrome-extension/) to learn more. 
```

**Review by rachelandrew — APPROVED** [review]
Just a tiny nit.

File: site/en/docs/devtools/recorder/index.md [hunk 2]
```diff
@@ -89,22 +89,7 @@ You can see the user flow replay on the page. The replay progress is shown in th
...
-   - Slow
-   - Very slow
-   - Extremely slow
-
-{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/40FyhHaVhPjQGatqUFij.png", alt="Slow replay.", width="800", height="525" %}
-
-{% Aside 'gotchas' %}
-You can use these slow replay options only in the **Recorder**. To add timeouts to the recording itself, see [Adjust timeouts for steps](/docs/devtools/recorder/reference/#adjust-timeout).
-{% endAside %}
+If you made a misclick during recording or something doesn't work, you can [debug your user flow](/docs/devtools/recorder/reference/#debug-user-flows): slow down its replay, set a breakpoint, and execute it step by step.
```
Comment (jecfish) [inline site/en/docs/devtools/recorder/index.md:92]: Move this paragraph up to before gotcha, it's easily missed.

**Review by jecfish — APPROVED** [review]
LGTM % nit

**Pull request merged**